File: string/cells-in-a-range-on-an-excel-sheet/cells-in-a-range-on-an-excel-sheet.ts
function cellsInRange(s: string): string[] {
  const colLow = s[0].charCodeAt(0);
  const colHigh = s[3].charCodeAt(0);
  const rowLow = s[1].charCodeAt(0);
  const rowHigh = s[4].charCodeAt(0);

  const ans: string[] = Array<string>((colHigh - colLow + 1) * (rowHigh - rowLow + 1));

  let i = 0;
  for (let col = colLow; col <= colHigh; col++) {
    for (let row = rowLow; row <= rowHigh; row++) {
      ans[i++] = String.fromCharCode(col) + String.fromCharCode(row);
    }
  }


  return ans;
};

File: string/cells-in-a-range-on-an-excel-sheet/cells-in-a-range-on-an-excel-sheet.go
package cellsinarangeonanexcelsheet

func cellsInRange(s string) []string {
	colLow := s[0]
	colHigh := s[3]
	rowLow := s[1]
	rowHigh := s[4]

	ansSize := (colHigh - colLow + 1) * (rowHigh - rowLow + 1)

	ans := make([]string, int(ansSize), int(ansSize))
	cel := make([]byte, 2, 2)

	i := 0
	for col := colLow; col <= colHigh; col++ {
		cel[0] = col
		for row := rowLow; row <= rowHigh; row++ {
			cel[1] = row
			ans[i] = string(cel)
			i++
		}
	}

	return ans
}
